media-sound/krecorder/krecorder-23.04.49.9999.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KFMIN=5.89.0
+QTMIN=5.15.0
+inherit ecm gear.kde.org
+
+DESCRIPTION="A convergent audio recording application for Plasma"
```

Comment:
Minor nit: We try to avoid filler words in packages maintained by kde proj, so I'd just start with `Convergent...`

Reply:
Fixed